backend/example.env

```diff
@@ -1,2 +0,0 @@
-MONGO_INITDB_ROOT_USERNAME='root'
```

Comment:
Can we keep this file as a template for anyone pulling down the repo - so that they know what a `.env` would look like?

Reply:
oh oops I thought I didnt delete that, will add back!